app/src/main/java/eu/kanade/tachiyomi/ui/main/MainActivity.kt

```diff
@@ -613,26 +610,38 @@ class MainActivity : BaseActivity() {
         internal val mpvVersions = MPVVersions()
 
         private var recordMPVLog = true
+    }
 
-        class RecordMPVVersion(context: Context) : MPVLib.LogObserver {
-            init {
-                MPVLib.create(context, "v")
-                MPVLib.addLogObserver(this@RecordMPVVersion)
-                MPVLib.init()
-            }
+    fun onLoggingComplete() {
+        lifecycleScope.launch(Dispatchers.IO) {
+            mpvVersions.trim()
+            MPVLib.removeLogObserver(recordMPVVersion)
+            MPVLib.destroy()
+            ready = true
```

Comment:
What does this line do?

Reply:
Why does it need to be explicitly called?, does it not auto turn off through the rest of the code?

Reply:
Imo would be better run it without using ready, since it has a different use already in the code. Don't want it to affect any other thing by accident